samples/tests/export_model_tables_classification_test.py

```diff
@@ -0,0 +1,42 @@
+# Generated code sample for google.cloud.aiplatform.DatasetServiceClient.get_dataset
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from samples import export_model_tables_classification_sample
+from google.cloud import storage
+
+PROJECT_ID = "ucaip-sample-tests"
```

Comment:
These should be env vars: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#external-resources

Reply:
All AI Platform samples now use env vars for `PROJECT_ID`